src/ui/page/booking/choose-date.html

```diff
@@ -41,22 +42,28 @@ <h2 class="heading-medium u-space-t10 heading--red-top">{ module.title || course
                                 </div>
                                 <div class="div-table__cell">{event.location}</div>
 
-                                <div class="div-table__cell div-table__cell--action">
-                                    <div class="multiple-choice multiple-choice--booking">
-                                        <input id="radio-{event.id}"
-                                        type="radio" name="selected-date"
-                                        class="multiple-choice__input"
-                                        checked="{selectedEventId === event.id ? 'checked' : ''}"
-                                        disabled="{selectedDate ? 'disabled' : ''}"
-                                        value="{event.id}"
-                                        >
-                                        <label for="radio-{event.id}" class="multiple-choice__label">
-                                            <span class="multiple-choice__spanner">
-                                                <span class="visuallyhidden">{$datetime.formatDate(event.date)}, {event.location}</span>
-                                            </span>
-                                        </label>
+                                {#if event.availability > 0}
+                                    <div class="div-table__cell">{event.availability} out of {event.capacity} places</div>
+                                    <div class="div-table__cell div-table__cell--action">
+                                        <div class="multiple-choice multiple-choice--booking">
+                                            <input id="radio-{event.id}"
+                                            type="radio" name="selected-date"
+                                            class="multiple-choice__input"
+                                            checked="{selectedEventId === event.id ? 'checked' : ''}"
+                                            disabled="{selectedDate ? 'disabled' : ''}"
+                                            value="{event.id}"
+                                            >
+                                            <label for="radio-{event.id}" class="multiple-choice__label">
+                                                <span class="multiple-choice__spanner">
+                                                    <span class="visuallyhidden">{$datetime.formatDate(event.date)}, {event.location}</span>
+                                                </span>
+                                            </label>
+                                        </div>
                                     </div>
-                                </div>
+                                {:else}
+                                    <div class="div-table__cell">fully booked</div>
```

Comment:
Can we capitalise this to sentence case?

Reply:
The design has it as lower case, but I could change it if you think that would be better